acme/acmego/main.go

```diff
@@ -70,6 +74,21 @@ func main() {
 		if event.Name == "" || event.Op != "put" {
 			continue
 		}
+
+		ws, err := acme.Windows()
+		if err != nil {
+			log.Fatal(err)
+		}
+		for _, w := range ws {
+			if w.ID != event.ID {
+				continue
+			}
+			if f := findFilter(w.Tag); f != "" {
```

Comment:
It looks like this does not handle tags with newlines in them. Will have to look into that.